docs/1-getting-started/creating-a-window.md

```diff
@@ -1 +1,65 @@
 # Creating a Window
+
+Creating a window using the Windows API library (a.k.a WINAPI32) can be made by following these steps:"
+
+## 1.) Fill a window class structure (WNDCLASS)
+
+```cpp
+WNDCLASS wndClass = {};
+wndClass.hInstance = hInstance; //hInstance from wWinMain/WinMain
+wndClass.lpfnWndProc = DefWindowProc; //function used to handle messages. We're currently using the default window message handling procedure. We will create our message handler function later...
```

Comment:
The contents of these comments should likely be explained before or after this code snippet. Besides, these comments are far too long to be on a single line.

Reply:
I think it is easier to read as it is right now to be honest...

Reply:
Most definitely not. This is how the current comments look when on the website:
![image](https://user-images.githubusercontent.com/43609023/154858948-d7423933-8232-4955-8d84-2f5186e989e8.png)

Scrolling horizontally is just annoying and does not give you a direct overview of all the contents.

Reply:
Oh, I will have this fixed shortly then.